test/exec/child.js

```diff
@@ -1,3 +1,5 @@
 // does nothing child process
 console.log("Child process.id: "+process.pid);
 console.log(" - - - - - - - - - - - - - - - - - - - - - - - ");
+// keep process alive for one minute
+setTimeout(function () {}, 60000);
```

Comment:
Ok. let's merge it and publish a new version. if we get a bug report we can go from there. 👍